srcpkgs/png++/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'png++'
+pkgname=png++
+version=0.2.9
+revision=1
+noarch=yes
+build_style=gnu-makefile
+make_build_target="docs"
```

Comment:
Just docs? Are you sure this actually installs a binary?
Pretty sure you want another target here.

Reply:
for reference: https://github.com/void-linux/void-packages/pull/3894